spec/shoryuken/middleware/server/auto_delete_spec.rb

```diff
@@ -55,8 +55,8 @@ def build_message
       expect(sqs_queue).to_not receive(:delete_messages)
 
       expect {
-        subject.call(TestWorker.new, queue, sqs_msg, sqs_msg.body) { raise }
-      }.to raise_error
+        subject.call(TestWorker.new, queue, sqs_msg, sqs_msg.body) { raise "Error" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.